Automattic/jetpack
# AI Assistant: add events for menu dropdowns

In order to monitor the tool usage, we'd need to check how discoverable it is

## Proposed changes:
This PR adds events upon dropdown menus showing

### Other information:


## Jetpack product discussion
pe4Cmx-236-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Go to the editor. Click on the AI Assistant toolbar icon. See the dops message on the console. 

See: p1706033825173369-slack-C054LN8RNVA
